
时间：2017-06-26 13:55:49,097 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：113733 

时间：2017-06-26 13:55:51,546 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：116182 

时间：2017-06-26 13:56:23,447 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：148084 

时间：2017-06-26 13:57:04,502 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：189138 

时间：2017-06-26 13:57:05,982 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：190618 

时间：2017-06-26 14:05:57,691 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：722328 

时间：2017-06-26 14:09:54,107 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：958743 

时间：2017-06-26 14:14:13,122 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：1217758 

时间：2017-06-26 14:14:17,318 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：1221954 

时间：2017-06-26 14:15:44,855 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：1309491 

时间：2017-06-26 14:20:03,427 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：1568063 

时间：2017-06-26 14:20:13,531 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：1578168 

时间：2017-06-26 14:20:21,307 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：1585943 

时间：2017-06-26 14:25:21,534 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：31731 
System.Data.SqlClient.SqlException (0x80131904): 在与 SQL Server 建立连接时出现与网络相关的或特定于实例的错误。未找到或无法访问服务器。请验证实例名称是否正确并且 SQL Server 已配置为允许远程连接。 (provider: Named Pipes Provider, error: 40 - 无法打开到 SQL Server 的连接) ---> System.ComponentModel.Win32Exception (0x80004005): 找不到网络路径。
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.Open()
   在 System.Data.Linq.SqlClient.SqlConnectionManager.UseConnection(IConnectionUser user)
   在 System.Data.Linq.SqlClient.SqlProvider.get_IsSqlCe()
   在 System.Data.Linq.SqlClient.SqlProvider.InitializeProviderMode()
   在 System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   在 System.Data.Linq.DataQuery`1.System.Collections.Generic.IEnumerable<T>.GetEnumerator()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 X.Data.x_dict.GetDictList(String code, String upval, DataClassesDataContext db) 位置 F:\HT\xue\src\Xc\Data\DataClasses.cs:行号 71
   在 X.Web.XFace.GetDictList(String code, String upval) 位置 F:\HT\xue\src\Web\X.Web\XFace.cs:行号 175
   在 X.Web.XFace.GetDict(String code, String val) 位置 F:\HT\xue\src\Web\X.Web\XFace.cs:行号 180
   在 X.App.Views._web.InitView() 位置 F:\HT\xue\src\Web\Xue\X.App\Views\_web.cs:行号 24
   在 X.Web.Views.View.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Views\View.cs:行号 94
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:53,State:0,Class:20

时间：2017-06-26 14:25:21,534 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：31731 
System.Data.SqlClient.SqlException (0x80131904): 在与 SQL Server 建立连接时出现与网络相关的或特定于实例的错误。未找到或无法访问服务器。请验证实例名称是否正确并且 SQL Server 已配置为允许远程连接。 (provider: Named Pipes Provider, error: 40 - 无法打开到 SQL Server 的连接) ---> System.ComponentModel.Win32Exception (0x80004005): 找不到网络路径。
   在 System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   在 System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   在 System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.Open()
   在 System.Data.Linq.SqlClient.SqlConnectionManager.UseConnection(IConnectionUser user)
   在 System.Data.Linq.SqlClient.SqlProvider.get_IsSqlCe()
   在 System.Data.Linq.SqlClient.SqlProvider.InitializeProviderMode()
   在 System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   在 System.Data.Linq.DataQuery`1.System.Collections.Generic.IEnumerable<T>.GetEnumerator()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 X.Data.x_dict.GetDictList(String code, String upval, DataClassesDataContext db) 位置 F:\HT\xue\src\Xc\Data\DataClasses.cs:行号 71
   在 X.Web.XFace.GetDictList(String code, String upval) 位置 F:\HT\xue\src\Web\X.Web\XFace.cs:行号 175
   在 X.Web.XFace.GetDict(String code, String val) 位置 F:\HT\xue\src\Web\X.Web\XFace.cs:行号 180
   在 X.App.Views._web.InitView() 位置 F:\HT\xue\src\Web\Xue\X.App\Views\_web.cs:行号 24
   在 X.Web.Views.View.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Views\View.cs:行号 94
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:53,State:0,Class:20

时间：2017-06-26 14:25:21,537 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：31734 
System.Data.SqlClient.SqlException (0x80131904): 在与 SQL Server 建立连接时出现与网络相关的或特定于实例的错误。未找到或无法访问服务器。请验证实例名称是否正确并且 SQL Server 已配置为允许远程连接。 (provider: Named Pipes Provider, error: 40 - 无法打开到 SQL Server 的连接) ---> System.ComponentModel.Win32Exception (0x80004005): 找不到网络路径。
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.Open()
   在 System.Data.Linq.SqlClient.SqlConnectionManager.UseConnection(IConnectionUser user)
   在 System.Data.Linq.SqlClient.SqlProvider.get_IsSqlCe()
   在 System.Data.Linq.SqlClient.SqlProvider.InitializeProviderMode()
   在 System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   在 System.Data.Linq.DataQuery`1.System.Collections.Generic.IEnumerable<T>.GetEnumerator()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 X.Data.x_dict.GetDictList(String code, String upval, DataClassesDataContext db) 位置 F:\HT\xue\src\Xc\Data\DataClasses.cs:行号 71
   在 X.Web.XFace.GetDictList(String code, String upval) 位置 F:\HT\xue\src\Web\X.Web\XFace.cs:行号 175
   在 X.Web.XFace.GetDict(String code, String val) 位置 F:\HT\xue\src\Web\X.Web\XFace.cs:行号 180
   在 X.App.Views._web.InitView() 位置 F:\HT\xue\src\Web\Xue\X.App\Views\_web.cs:行号 24
   在 X.Web.Views.View.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Views\View.cs:行号 94
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:53,State:0,Class:20

时间：2017-06-26 15:02:18,988 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：2249185 

时间：2017-06-26 15:02:23,050 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：2253247 

时间：2017-06-26 15:02:33,054 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：2263251 

时间：2017-06-26 15:03:13,107 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：2303304 

时间：2017-06-26 15:13:38,771 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：52217 
NVelocity.Exception.MethodInvocationException: Invocation of method 'GetQues' in  X.App.Views.paper.edit threw exception System.NotSupportedException : 不能在查询运算符(Contains 运算符除外)的 LINQ to SQL 实现中使用本地序列。 ---> System.NotSupportedException: 不能在查询运算符(Contains 运算符除外)的 LINQ to SQL 实现中使用本地序列。
   在 System.Data.Linq.SqlClient.QueryConverter.CoerceToSequence(SqlNode node)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitAggregate(Expression sequence, LambdaExpression lambda, SqlNodeType aggType, Type returnType)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitSequenceOperatorCall(MethodCallExpression mc)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitMethodCall(MethodCallExpression mc)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitInner(Expression node)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitExpression(Expression exp)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitBinary(BinaryExpression b)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitInner(Expression node)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitExpression(Expression exp)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitBinary(BinaryExpression b)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitInner(Expression node)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitExpression(Expression exp)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitConditional(ConditionalExpression cond)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitInner(Expression node)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitExpression(Expression exp)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitNew(NewExpression qn)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitInner(Expression node)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitSelect(Expression sequence, LambdaExpression selector)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitSequenceOperatorCall(MethodCallExpression mc)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitMethodCall(MethodCallExpression mc)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitInner(Expression node)
   在 System.Data.Linq.SqlClient.QueryConverter.ConvertOuter(Expression node)
   在 System.Data.Linq.SqlClient.SqlProvider.BuildQuery(Expression query, SqlNodeAnnotations annotations)
   在 System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   在 System.Data.Linq.DataQuery`1.System.Collections.Generic.IEnumerable<T>.GetEnumerator()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 X.App.Views.paper.edit.GetQues(String k) 位置 F:\HT\xue\src\Web\Xue\X.App\Views\paper\edit.cs:行号 38
   --- 内部异常堆栈跟踪的结尾 ---
   在 X.Web.Views.View.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Views\View.cs:行号 108
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-26 15:15:03,300 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：15052 
NVelocity.Exception.MethodInvocationException: Invocation of method 'GetQues' in  X.App.Views.paper.edit threw exception System.NotSupportedException : 不能在查询运算符(Contains 运算符除外)的 LINQ to SQL 实现中使用本地序列。 ---> System.NotSupportedException: 不能在查询运算符(Contains 运算符除外)的 LINQ to SQL 实现中使用本地序列。
   在 System.Data.Linq.SqlClient.QueryConverter.CoerceToSequence(SqlNode node)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitAggregate(Expression sequence, LambdaExpression lambda, SqlNodeType aggType, Type returnType)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitSequenceOperatorCall(MethodCallExpression mc)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitMethodCall(MethodCallExpression mc)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitInner(Expression node)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitExpression(Expression exp)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitBinary(BinaryExpression b)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitInner(Expression node)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitExpression(Expression exp)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitBinary(BinaryExpression b)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitInner(Expression node)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitExpression(Expression exp)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitConditional(ConditionalExpression cond)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitInner(Expression node)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitExpression(Expression exp)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitNew(NewExpression qn)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitInner(Expression node)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitSelect(Expression sequence, LambdaExpression selector)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitSequenceOperatorCall(MethodCallExpression mc)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitMethodCall(MethodCallExpression mc)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitInner(Expression node)
   在 System.Data.Linq.SqlClient.QueryConverter.ConvertOuter(Expression node)
   在 System.Data.Linq.SqlClient.SqlProvider.BuildQuery(Expression query, SqlNodeAnnotations annotations)
   在 System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   在 System.Data.Linq.DataQuery`1.System.Collections.Generic.IEnumerable<T>.GetEnumerator()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 X.App.Views.paper.edit.GetQues(String k) 位置 F:\HT\xue\src\Web\Xue\X.App\Views\paper\edit.cs:行号 38
   --- 内部异常堆栈跟踪的结尾 ---
   在 X.Web.Views.View.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Views\View.cs:行号 108
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-26 15:26:43,573 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：2866 
NVelocity.Exception.MethodInvocationException: Invocation of method 'GetQues' in  X.App.Views.paper.edit threw exception System.NotSupportedException : 不能在查询运算符(Contains 运算符除外)的 LINQ to SQL 实现中使用本地序列。 ---> System.NotSupportedException: 不能在查询运算符(Contains 运算符除外)的 LINQ to SQL 实现中使用本地序列。
   在 System.Data.Linq.SqlClient.QueryConverter.CoerceToSequence(SqlNode node)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitAggregate(Expression sequence, LambdaExpression lambda, SqlNodeType aggType, Type returnType)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitSequenceOperatorCall(MethodCallExpression mc)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitMethodCall(MethodCallExpression mc)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitInner(Expression node)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitExpression(Expression exp)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitBinary(BinaryExpression b)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitInner(Expression node)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitExpression(Expression exp)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitBinary(BinaryExpression b)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitInner(Expression node)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitExpression(Expression exp)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitConditional(ConditionalExpression cond)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitInner(Expression node)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitExpression(Expression exp)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitNew(NewExpression qn)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitInner(Expression node)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitSelect(Expression sequence, LambdaExpression selector)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitSequenceOperatorCall(MethodCallExpression mc)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitMethodCall(MethodCallExpression mc)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitInner(Expression node)
   在 System.Data.Linq.SqlClient.QueryConverter.ConvertOuter(Expression node)
   在 System.Data.Linq.SqlClient.SqlProvider.BuildQuery(Expression query, SqlNodeAnnotations annotations)
   在 System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   在 System.Data.Linq.DataQuery`1.System.Collections.Generic.IEnumerable<T>.GetEnumerator()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 X.App.Views.paper.edit.GetQues(String k) 位置 F:\HT\xue\src\Web\Xue\X.App\Views\paper\edit.cs:行号 37
   --- 内部异常堆栈跟踪的结尾 ---
   在 X.Web.Views.View.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Views\View.cs:行号 108
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-26 15:26:48,743 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：8035 
NVelocity.Exception.MethodInvocationException: Invocation of method 'GetQues' in  X.App.Views.paper.edit threw exception System.NotSupportedException : 不能在查询运算符(Contains 运算符除外)的 LINQ to SQL 实现中使用本地序列。 ---> System.NotSupportedException: 不能在查询运算符(Contains 运算符除外)的 LINQ to SQL 实现中使用本地序列。
   在 System.Data.Linq.SqlClient.QueryConverter.CoerceToSequence(SqlNode node)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitAggregate(Expression sequence, LambdaExpression lambda, SqlNodeType aggType, Type returnType)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitSequenceOperatorCall(MethodCallExpression mc)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitMethodCall(MethodCallExpression mc)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitInner(Expression node)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitExpression(Expression exp)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitBinary(BinaryExpression b)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitInner(Expression node)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitExpression(Expression exp)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitBinary(BinaryExpression b)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitInner(Expression node)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitExpression(Expression exp)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitConditional(ConditionalExpression cond)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitInner(Expression node)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitExpression(Expression exp)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitNew(NewExpression qn)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitInner(Expression node)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitSelect(Expression sequence, LambdaExpression selector)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitSequenceOperatorCall(MethodCallExpression mc)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitMethodCall(MethodCallExpression mc)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitInner(Expression node)
   在 System.Data.Linq.SqlClient.QueryConverter.ConvertOuter(Expression node)
   在 System.Data.Linq.SqlClient.SqlProvider.BuildQuery(Expression query, SqlNodeAnnotations annotations)
   在 System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   在 System.Data.Linq.DataQuery`1.System.Collections.Generic.IEnumerable<T>.GetEnumerator()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 X.App.Views.paper.edit.GetQues(String k) 位置 F:\HT\xue\src\Web\Xue\X.App\Views\paper\edit.cs:行号 37
   --- 内部异常堆栈跟踪的结尾 ---
   在 X.Web.Views.View.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Views\View.cs:行号 108
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-26 15:28:17,995 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：97287 
NVelocity.Exception.MethodInvocationException: Invocation of method 'GetQues' in  X.App.Views.paper.edit threw exception System.NotSupportedException : 不能在查询运算符(Contains 运算符除外)的 LINQ to SQL 实现中使用本地序列。 ---> System.NotSupportedException: 不能在查询运算符(Contains 运算符除外)的 LINQ to SQL 实现中使用本地序列。
   在 System.Data.Linq.SqlClient.QueryConverter.CoerceToSequence(SqlNode node)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitAggregate(Expression sequence, LambdaExpression lambda, SqlNodeType aggType, Type returnType)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitSequenceOperatorCall(MethodCallExpression mc)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitMethodCall(MethodCallExpression mc)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitInner(Expression node)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitExpression(Expression exp)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitBinary(BinaryExpression b)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitInner(Expression node)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitExpression(Expression exp)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitBinary(BinaryExpression b)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitInner(Expression node)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitExpression(Expression exp)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitConditional(ConditionalExpression cond)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitInner(Expression node)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitExpression(Expression exp)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitNew(NewExpression qn)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitInner(Expression node)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitSelect(Expression sequence, LambdaExpression selector)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitSequenceOperatorCall(MethodCallExpression mc)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitMethodCall(MethodCallExpression mc)
   在 System.Data.Linq.SqlClient.QueryConverter.VisitInner(Expression node)
   在 System.Data.Linq.SqlClient.QueryConverter.ConvertOuter(Expression node)
   在 System.Data.Linq.SqlClient.SqlProvider.BuildQuery(Expression query, SqlNodeAnnotations annotations)
   在 System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   在 System.Data.Linq.DataQuery`1.System.Collections.Generic.IEnumerable<T>.GetEnumerator()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 X.App.Views.paper.edit.GetQues(String k) 位置 F:\HT\xue\src\Web\Xue\X.App\Views\paper\edit.cs:行号 37
   --- 内部异常堆栈跟踪的结尾 ---
   在 X.Web.Views.View.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Views\View.cs:行号 108
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-26 15:38:07,470 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：138240 

时间：2017-06-26 15:38:07,472 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：138242 

时间：2017-06-26 15:38:07,479 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：138249 

时间：2017-06-26 16:18:22,747 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：2127552 
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 X.App.Views.paper.create.InitDict()
   在 X.Web.Views.View.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Views\View.cs:行号 95
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-26 16:18:28,058 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：2132863 
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 X.App.Views.paper.create.InitDict()
   在 X.Web.Views.View.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Views\View.cs:行号 95
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-26 16:18:48,912 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：2153718 
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 X.App.Views.paper.create.InitDict()
   在 X.Web.Views.View.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Views\View.cs:行号 95
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-26 16:18:57,104 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：2161909 
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 X.App.Views.paper.create.InitDict()
   在 X.Web.Views.View.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Views\View.cs:行号 95
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-26 16:19:06,483 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：2171289 
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 X.App.Views.paper.create.InitDict()
   在 X.Web.Views.View.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Views\View.cs:行号 95
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-26 16:25:01,109 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：2525914 
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 X.App.Views.paper.create.InitDict()
   在 X.Web.Views.View.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Views\View.cs:行号 95
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-26 16:25:04,312 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：2529117 
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 X.App.Views.paper.create.InitDict()
   在 X.Web.Views.View.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Views\View.cs:行号 95
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-26 16:43:12,394 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：311 

时间：2017-06-26 16:45:43,978 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：8412 

时间：2017-06-26 16:45:55,556 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"验证码不正确 ","code":"0x0007"} 
时间：19991 

时间：2017-06-26 17:14:06,166 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"缺少参数 筛选条件","code":"0x0003"} 
时间：319979 

时间：2017-06-26 17:14:08,515 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：322328 
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 X.App.Views.paper.edit.InitDict()
   在 X.Web.Views.View.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Views\View.cs:行号 95
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-26 17:14:26,412 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：340225 
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 X.App.Views.paper.create.InitDict()
   在 X.Web.Views.View.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Views\View.cs:行号 95
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-26 17:35:10,231 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：63841 

时间：2017-06-26 17:35:45,881 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：99491 

时间：2017-06-26 17:35:50,785 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：104394 

时间：2017-06-26 17:36:32,620 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：146230 
